include/view.h

```diff
@@ -68,6 +68,8 @@ enum view_wants_focus {
 };
 
 struct view;
+struct wlr_output;
+struct wlr_output_layout;
```

Comment:
Leftovers from a rebase or similar?
I didn't spot any `wlr_output` nor `wlr_output_layout` in `view.h` at least.

Reply:
Hmm yes, these were probably for `xwayland_adjust_usable_area` which is in xwayland.h now.